src/reviews/views.py

```diff
@@ -0,0 +1,45 @@
+from .models import Review
+from .serializers import ReviewSerializer
+from rest_framework.permissions import AllowAny, IsAuthenticated
+from middlewares.permissions import IsOwner
+from rest_framework.viewsets import ModelViewSet
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+
+
+@api_view(['GET'])
+def LikeReview(request, pk):
+    queryset = Review.objects.get(pk=pk)
+    if request.user in queryset.like.all():
```

Comment:
find like theo user luôn, xem có record nào với user đấy ko,
chứ ko làm kiểu `in` này